man/systemd.network.xml

```diff
@@ -1004,6 +1004,25 @@ DuplicateAddressDetection=none</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MulticastIGMPVersion=</varname></term>
+        <listitem>
+          <para>Configure IPv4 Multicast IGMP Version. If not configure not change the IPV4 Multicast IGMP protocol version.
```

Comment:
`Configure` -> `Configures`

Maybe, better to mention the sysctl path. E.g.
`Configures IPv4 Multicast IGMP Version to be used, and controls the value of <filename>/proc/sys/net/ipv4/conf/<replaceable>INTERFACE</replaceable>/force_igmp_version</filename>.`

`If not configure not change the IPV4 Multicast IGMP protocol version.` -> `If not specified, the IPv4 Multicast IGMP protocol version for the interface will be unchanged.`

Reply:
done